infra/src/dsl/types/output-types.ts

```diff
@@ -181,7 +181,7 @@ export type HelmValueFile = {
 }
 export type LocalrunValueFile = {
   services: Services<LocalrunService>
-  mocks: Services<LocalrunService>
+  mocks: string
```

Comment:
Is this so that new services can be added without changing the interface?

Reply:
No, I am just adjusting the output for the mocks section - it is a command string to be copy/pasted into the terminal.